tools/behave_rule.bzl

```diff
@@ -61,18 +61,49 @@ def _rule_implementation(ctx):
            fi
            DIRECTORY=$(ls ./grakn_distribution)
            echo Successfully unarchived Grakn distribution.
+
+           RND=20001
+           while [ $RND -gt 20000 ]  # Guarantee fair distribution of random ports
+           do
+             RND=$RANDOM
+           done
+           PORT=$((40000 + $RND))
```

Comment:
`$RANDOM` returns a random integer in the range `[0..32767]`. This means that when taking `$RANDOM % 20000`, the result is not a fair distribution - for example, `0` is twice as likely as `15000` to be picked.

To mitigate this issue and minimise clashes, we reroll until we get a random number in the range `[0..20000]`.